autoscheduler/frontend/src/components/LandingPage/About/About.css

```
@@ -0,0 +1,41 @@
+/* The div that holds the link to open the dialog on the home page */
+.dialog-link {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 32px;
+    /* text-decoration: underline;
+    text-decoration-color: #500; */

```

```suggestion
    justify-content: center;
    padding-top: 32px;
```

`align-items` isn't needed since there's only one child